robot/basestation/static/js/elements/navigation/localization.js

```diff
@@ -0,0 +1,129 @@
+let rover_heading = 0
+let rover_x = 0
+let rover_y = 0
+let zoom_value = 1
+let MAX_ZOOM = 2
+let MIN_ZOOM = 0.25
+let canvas_pos
+let keys = { //javascript keycodes
+    'r': 82,
+    'w': 87,
+    'a': 65,
+    's': 83,
+    'd': 68
+}
+
+let base_lat
+let base_long
+$(document).ready(() => {
+    // the elements might not be initialized yet, must test
+    base_lat = $('#antenna-latitude').text()
+    base_long = $('#antenna-longitude').text()
+})
+
+function setup() {
+    let canvas = createCanvas(450, 300)
+    canvas.parent('#canvasContainer')
+    canvas_pos = $('#defaultCanvas0').offset()
+    angleMode(DEGREES)
+}
+
+function draw() {
+    // all of the manual controls
+    if (keyIsDown(keys['r'])) { //reset position
```

Comment:
there's probably another way to do this, have you looked at the rest of our codebase?

Reply:
this is just for testing purposes and won't be used during operation